### DISCUSSION TITLE
alist在电脑重启后报错，后台管理重新加载全部后正常

### DISCUSSION THREAD

**Discussion by hyyxiaoluo** [AlistGo/alist#3936] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/102205671/227418979-c1dadd00-d35d-450c-bd32-d02b20ccd383.png)
请问这是什么原因造成的，翻译显示驱动禁止啥的，是因为自己的电脑的一些驱动问题吗，主要是我的token也没问题啊，获取好几次了还是这样，重启后就给我出现这，大佬们帮帮忙，求解！

**Comment by sunnywangzi** [comment, marked as answer]
挂载的是什么？可以放一下运行日志

**Reply by hyyxiaoluo** [reply]
[GIN] 2023/03/24 - 15:04:44 | 401 |            0s |       127.0.0.1 | PROPFIND  "/dav/"
[GIN] 2023/03/24 - 15:04:44 | 207 |       847.1µs |       127.0.0.1 | PROPFIND  "/dav/"
[GIN] 2023/03/24 - 15:04:44 | 200 |       395.7µs |       127.0.0.1 | OPTIONS  "/dav/"
[GIN] 2023/03/24 - 15:04:44 | 401 |            0s |       127.0.0.1 | PROPFIND  "/dav/"
[GIN] 2023/03/24 - 15:04:44 | 207 |            0s |       127.0.0.1 | PROPFIND  "/dav/"
[GIN] 2023/03/24 - 15:04:59 | 207 |       509.4µs |       127.0.0.1 | PROPFIND  "/dav/"
[GIN] 2023/03/24 - 15:06:04 | 200 |            0s |       127.0.0.1 | GET      "/"
[GIN] 2023/03/24 - 15:06:04 | 302 |       121.7µs |       127.0.0.1 | GET      "/favicon.ico"
[GIN] 2023/03/24 - 15:06:05 | 200 |      1.0031ms |       127.0.0.1 | GET      "/api/public/settings"
[GIN] 2023/03/24 - 15:06:05 | 200 |       440.5µs |       127.0.0.1 | GET      "/api/me"
[GIN] 2023/03/24 - 15:06:05 | 200 |       756.7µs |       127.0.0.1 | POST     "/api/fs/get"
[GIN] 2023/03/24 - 15:06:05 | 200 |      1.0635ms |       127.0.0.1 | POST     "/api/fs/list"
[31mERRO[0m[2023-03-24 15:06:17] ForbiddenDriveNotValid:not valid driveId
failed to list objs
github.com/alist-org/alist/v3/internal/op.List.func1
	/source/internal/op/fs.go:120
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall.func2
	/source/pkg/singleflight/singleflight.go:193
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall
	/source/pkg/singleflight/singleflight.go:195
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).Do
	/source/pkg/singleflight/singleflight.go:108
github.com/alist-org/alist/v3/internal/op.List
	/source/internal/op/fs.go:117
github.com/alist-org/alist/v3/internal/fs.list
	/source/internal/fs/list.go:25
github.com/alist-org/alist/v3/internal/fs.List
	/source/internal/fs/fs.go:17
github.com/alist-org/alist/v3/server/handles.FsList
	/source/server/handles/fsread.go:80
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/alist-org/alist/v3/server/middlewares.Auth
	/source/server/middlewares/auth.go:67
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/alist-org/alist/v3/server/middlewares.StoragesLoaded
	/source/server/middlewares/check.go:14
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/recovery.go:102
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/logger.go:240
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/gin.go:620
github.com/gin-gonic/gin.(*Engine).ServeHTTP
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/gin.go:576
net/http.serverHandler.ServeHTTP
	/usr/local/go/src/net/http/server.go:2936
net/http.(*conn).serve
	/usr/local/go/src/net/http/server.go:1995
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1598 
[31mERRO[0m[2023-03-24 15:06:17] failed list /YYの资源（open）: ForbiddenDriveNotValid:not valid driveId
failed to list objs
github.com/alist-org/alist/v3/internal/op.List.func1
	/source/internal/op/fs.go:120
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall.func2
	/source/pkg/singleflight/singleflight.go:193
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall
	/source/pkg/singleflight/singleflight.go:195
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).Do
	/source/pkg/singleflight/singleflight.go:108
github.com/alist-org/alist/v3/internal/op.List
	/source/internal/op/fs.go:117
github.com/alist-org/alist/v3/internal/fs.list
	/source/internal/fs/list.go:25
github.com/alist-org/alist/v3/internal/fs.List
	/source/internal/fs/fs.go:17
github.com/alist-org/alist/v3/server/handles.FsList
	/source/server/handles/fsread.go:80
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/alist-org/alist/v3/server/middlewares.Auth
	/source/server/middlewares/auth.go:67
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/alist-org/alist/v3/server/middlewares.StoragesLoaded
	/source/server/middlewares/check.go:14
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/recovery.go:102
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/gin-gonic/gin.LoggerWithConfig.func1
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/logger.go:240
github.com/gin-gonic/gin.(*Context).Next
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/context.go:174
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/gin.go:620
github.com/gin-gonic/gin.(*Engine).ServeHTTP
	/go/pkg/mod/github.com/gin-gonic/gin@v1.9.0/gin.go:576
net/http.serverHandler.ServeHTTP
	/usr/local/go/src/net/http/server.go:2936
net/http.(*conn).serve
	/usr/local/go/src/net/http/server.go:1995
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1598
failed get objs 
[GIN] 2023/03/24 - 15:06:17 | 200 |    769.8568ms |       127.0.0.1 | POST     "/api/fs/list"
我把之前的日志删了重启电脑复现了一下，大佬帮忙看下谢谢了

**Reply by sunnywangzi** [reply]
挂载的是阿里网盘？试试用open驱动

**Reply by hyyxiaoluo** [reply]
就是open

**Reply by sunnywangzi** [reply]
我这边测试是正常的，我猜测应该是电脑刚开机时网络还没有连接，alist就去尝试加载驱动（此时没有网络当然加载失败），后面在管理页面重新加载后（此时网络正常），可以尝试开机后延迟几秒在启动alist。（我之前遇到过就是这么解决的

**Reply by hyyxiaoluo** [reply]
好的我试一试，谢谢你呀，貌似是的，之前在启动里添加的脚本快捷方式启动有时候开机radrive都启动了他还没启动，然后radrive就弹连不上，所以我设置的任务计划来启动脚本，貌似可能太快了，网跟不上，我等下重启试试

**Reply by sunnywangzi** [reply]
好的，如果解决了，可以把上面的回答设置成答案

**Reply by hyyxiaoluo** [reply]
确实是这个原因，换回原来的快捷方式脚本启动正常了
